### PR TITLE
adds updated wics-csc event

### PR DESCRIPTION
**Pull request by bdamerac**

### CODE REVIEW

**Comment by bdamerac** [comment]
Should I have created a new file instead?

**Comment by evykassirer** [comment]
this seems reasonable to me!

**Comment by arshiamufti** [comment]
I think a new file would be better since we'd have a record of the cancelled event on our website for anyone who was looking for bonfire event details.

**Comment by evykassirer** [comment]
^true ++

File: content/S2016/S16-event-wics-csc.md [hunk 1]
```diff
@@ -7,8 +7,8 @@ Author: Clare Ng
 Summary: We're having a bonfire with the CSC! There will be ice cream and marshmellows.
 
 **This event has been cancelled due to fire ban for KW and surrounding areas 
-due to dry weather conditions. We're trying to organize an alternate event, so
-we'll keep you posted!**
+due to dry weather conditions. Details about updated event can be found**
+[here]({filename}/content/S2016/S16-updated-event-wics-csc.md).
```
Comment (evykassirer) [inline content/S2016/S16-event-wics-csc.md]: can you make this bolded too? (move the `**` to after the link)

File: content/S2016/S16-updated-event-wics-csc.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+title: WiCS & CSC Go Outside
+Date: 2016-07-08
+Category: Events
+Tags: social
+Slug: wics-csc-s16
+Author: Bhargavi Dameracharla
+Summary: Join WiCS and CSC pinic + outdoor games event
+
+WiCS and CSC are hosting a picnic and outdoor games event at the BMH green.
+Be a part of the fun. There will be food, frisbees, maybe waterguns.
```
Comment (evykassirer) [inline content/S2016/S16-updated-event-wics-csc.md]: can you put exclamation mark after fun? 😄 and an "and" before "maybe waterguns"

**Pull request merged**